CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 # Datadog AppSec for PHP Release
 
+### v0.5.0
+#### Fixes
+ - ([#120](https://github.com/DataDog/dd-appsec-php/pull/120)) Return error response in helper when incoming message can't be adequately handled
+ - ([#124](https://github.com/DataDog/dd-appsec-php/pull/124)) Avoid creating log file as root
+ - ([#130](https://github.com/DataDog/dd-appsec-php/pull/130)) Reset context on shutdown
+ - ([#132](https://github.com/DataDog/dd-appsec-php/pull/132)) Handle errors on `request_shutdown`
+ - ([#170](https://github.com/DataDog/dd-appsec-php/pull/170)) Avoid regenerating ip when multiple headers are already present
+
+#### Additions
+ - ([#114](https://github.com/DataDog/dd-appsec-php/pull/114)) Add zai config
+ - ([#128](https://github.com/DataDog/dd-appsec-php/pull/128)) Replace `actor.ip` with `http.client_ip`
+ - ([#151](https://github.com/DataDog/dd-appsec-php/pull/151)) PHP 8.2RC support
+ - ([#155](https://github.com/DataDog/dd-appsec-php/pull/155)) Generate IP on appsec
```

Comment:
This got added to the previous version as well

Reply:
The history is a bit of a mess because of all the patch releases, cherry-picks and such...